mlir/lib/Dialect/Tensor/IR/TensorOps.cpp

```diff
@@ -4143,9 +4145,12 @@ bool PackOp::isLikePad() {
 }
 
 OpFoldResult PackOp::fold(FoldAdaptor adaptor) {
+  std::optional<Attribute> paddingValue;
+  if (adaptor.getPaddingValue())
+    paddingValue = adaptor.getPaddingValue();
```

Comment:
Nit: we should avoid the double call to `getPaddingValue()` here: this accessor is far from free (there is a loop hidden behind it)